flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/BoundedBlockingSubpartition.java

```diff
@@ -145,6 +147,11 @@ private void flushCurrentBuffer() throws IOException {
 		}
 	}
 
+	@Override
+	public Collection<Buffer> getInflightingBuffers() {
+		return Collections.emptyList();
```

Comment:
I'm not sure what's your final plan how to deal with `BoundedBlockingSubpartition`, but shouldn't we leave a `//TODO: ` comment here? Or throwing an `UnsupportedOperationException`?

Reply:
For batch jobs we do not have checkpoint mechanism atm, so this method would never be called actually. `UnsupportedOperationException` would be better.